priv/templates/boot_win.eex

```diff
@@ -2,6 +2,8 @@
 ::
 :: Check the :usage section for information about what this script provides
 
+@echo off
```

Comment:
In the script practically every line starts with @, which causes the line to not be output on the terminal. This line makes it the default behavior so all the @ aren't necessary anymore.